Update de.html L1250-1425

randomcmd opened the pull request.

randomcmd committed:
1250 untill 1425

herrhundt commented on de.html:
```diff
@@ -1255,17 +1255,17 @@
 </words>
 <words x=30 y=130 w=240 no-bg>
-	So kannst du die Fakten mit etwas verknüpfen was <i>du</i> weisst oder magst.
+	So kannst du die Fakten mit etwas verknüpfen was <i>du</i> weisst und <i>du</i> magst.
```
Vorschlag: 
So kannst du die Fakten mit etwas verknüpfen, was <i> du </i> weißt und magst.

herrhundt commented on de.html:
```diff
@@ -1342,11 +1342,10 @@
 <img src="pics/fcard_frontonly.png" style="width:100%; height:100%; position:absolute; top:0; left:0;"/>
 </panel>
-
+<!--"Was willst du lernen" finde ich hier nicht-->
```
Der Text der Karten ist nicht in dieser Datei zu finden ;-)

herrhundt commented on de.html:
```diff
@@ -1356,19 +1355,19 @@
 </words>
 <words x=129 y=283 w=200 style="text-align:left" no-bg class="comic_text">
-	alles was du interessant findest, egal wo!
+	Alles was du interessant findest, egal wo!
```
Alles, was du interessant findest, egal wo!

herrhundt commented on de.html:
```diff
@@ -1356,19 +1355,19 @@
 </words>
 <words x=80 y=380 w=380 h=30>
-	Schreibe <i>deine</i> Antwort jetzt auf die Rückseite:
+	Jetzt schreibe <i>deine</i> Antwort jetzt auf die Rückseite:
```
Doppelt jetzt, Verbesserung: 
Schreibe <i>deine</i> Antwort jetzt auf die Rückseite:

herrhundt commented on de.html:
```diff
@@ -1399,23 +1398,23 @@
-	Das könnte zu philosophisch sein, daher hier ein
-	konkretes Beispiel für ein <i>Warum</i> zu einem <i>Was</i>:
+	Das könnte dir jetzt zu philosophisch sein, deshalb hier
+	spezielles Beispiel für dem <i>Warum</i> hinter einem <i>Was</i>:
```
Wort fehlt: ein 
ein spezielles (...) und heißt es nicht für **das** Warum?

herrhundt commented on de.html:
```diff
@@ -1399,23 +1398,23 @@
 	<b>WAS:</b> Programmierung
 	<br>
-	<b>WARUM:</b> Geld zu verdienen um dir Essen leisten zu können
+	<b>WARUM:</b> um Geld zu verdienen um dir Essen leisten zu können
```
klingt irgendwie unschön

its-nico replied:
```suggestion
	<b>WARUM:</b> damit du Geld zu verdienst, um dir Essen leisten zu können
```

herrhundt commented on de.html:
```diff
@@ -1399,23 +1398,23 @@
 	<b>WAS:</b> was auch immer dich interessiert
 	<br>
-	<b>WARUM:</b> Neugierde, ihrer selbst willen!
+	<b>WARUM:</b> Neugier, aus intrinischem Willen!
```
das klingt super!

randomcmd committed:
Update de.html 1250-1500 [Waiting for approval]

herrhundt committed:
Update de.html

Co-authored-by: its-nico <65342313+its-nico@users.noreply.github.com>

its-nico committed:
Udate de.html L1264-1358/1373-1398/1405-1409/1415-1422/1454-1496

Diese Zeilen konnte ich in Adrians pull request nicht kommentieren, da sie eingeklappt waren, und das dann anscheinend nicht funktioniert. In diesen Zeilen habe ich ein paar Dinge verbessert bzw. andere Formulierungen vorgeschlagen

noahdyj committed:
Update de.html L1364

noahdyj committed:
Update de.htlm L1405

noahdyj committed:
Update de.html L1412

its-nico commented on de.html:
```diff
@@ -1399,23 +1398,23 @@
+	<b>WARUM:</b> um mit Freunden, Familie und Geliebten in ihrer Muttersprache zu reden
 </words>
 <words x=130 y=210 w=350 style="text-align:left" no-bg class="comic_text">
 	<b>WAS:</b> Programmierung
```
```suggestion
	<b>WAS:</b> eine Programmiersprache
```

its-nico committed:
Update de.html

its-nico committed:
Merge branch 'randomcmd-patch-2' into randomcmd-patch-3

its-nico committed:
Merge pull request #25 from herrhundt/its-nico-patch-3

Udate de.html L1264-1358/1373-1398/1405-1409/1415-1422/1454-1496

its-nico committed:
Update de.html

Co-authored-by: herrhundt <59929867+herrhundt@users.noreply.github.com>

its-nico committed:
Update de.html

its-nico committed:
Update de.html

herrhundt committed:
Update de.html

Co-authored-by: its-nico <65342313+its-nico@users.noreply.github.com>

herrhundt committed:
Update de.html

Co-authored-by: its-nico <65342313+its-nico@users.noreply.github.com>

herrhundt committed:
Update de.html

Co-authored-by: its-nico <65342313+its-nico@users.noreply.github.com>

herrhundt committed:
Update de.html

herrhundt committed:
Merge pull request #15 from herrhundt/randomcmd-patch-3

Update de.html 1250-1500 [Waiting for approval]

herrhundt committed:
Merge branch 'master' into randomcmd-patch-2
